app/assets/javascripts/modules/moj.countable.js

```diff
@@ -0,0 +1,51 @@
+var Countable = require('../vendor/Countable');
+
+module.exports = (function() {
+
+  var countable = {};
+
+  countable.init = function(area) {
+
+    var max = $(area).attr('maxlength'),
+      hint = $(area).prev('.form-hint'),
+      txt = hint.text();
+
+    // IE!
+    if (!('maxLength' in area)) {
+      max = area.attributes.maxLength.value;
+      area.onkeypress = function () {
+          if (this.value.length >= max) return false;
+      };
+    }
+
+    Countable.live(area, callback);
+
+    function callback(counter) {
+      hint.html(txt + ' (' + remainder(counter.all) + ' characters remaining)');
+    };
+
+    function remainder(count){
+      var remaining = parseInt(max - count, 10);
+
+      if(count === 0) {
+        return max;
+      } else if(remaining >= max) {
+        return '0 ';
+      } else {
+        return remaining;
+      }
+    }
+  };
+
+  if($('textarea').length){
```

Comment:
@dan-palmer thats the intention. it stops the rest of the script from running if textareas aren't present in the page